week7/bmi_calculator_react/01_acceptance_testing_in_react.md

```diff
@@ -30,6 +30,15 @@ Once we run cypress for first time, it will create scaffolded folder, with a lot
 
 Our tests are stored in `Integration` folder and `fixtures` folder will contain JSON files, which we will introduce during the course of this document.
 
+Configure our baseUrl in `cypress.json` file:
```

Comment:
format as `baseUrl` (as code)  Also, what is `baseUrl` - the student needs an explanation.

Reply:
done